lib/oauthutil/oauthutil.go

```diff
@@ -267,9 +266,10 @@ func (ts *TokenSource) Token() (*oauth2.Token, error) {
 			if ts.reReadToken() {
 				changed = true
 			} else if ts.token.RefreshToken == "" {
-				return nil, fserrors.FatalError(
-					fmt.Errorf("token expired and there's no refresh token - manually refresh with \"rclone config reconnect %s:\"", ts.name),
-				)
+				//Box authentication OAuth2.0 with JWT does not provide refresh tokens
```

Comment:
Can we tell if we are using JWT here and if so not give this error, otherwise give the error? As changing this code will affect all the backends which use Oauth which is nearly all of them.

Alternatively we could return a special error which the box backend can detect or make a config in the oauth setup.

Reply:
Any thoughts on this @aselafernando - I don't want to affect all the oauth using backends in this way.